src/presign/round_three.rs

```diff
@@ -20,25 +20,88 @@ use crate::{
         Proof, ProofContext,
     },
 };
-use k256::{elliptic_curve::PrimeField, Scalar};
+use k256::{
+    elliptic_curve::{subtle::CtOption, PrimeField},
+    Scalar,
+};
 use libpaillier::unknown_order::BigNumber;
 use merlin::Transcript;
 use serde::{Deserialize, Serialize};
-use std::fmt::Debug;
+use std::{
+    fmt::Debug,
+    ops::{AddAssign, Deref},
+};
 use tracing::error;
 use zeroize::ZeroizeOnDrop;
+//use libpaillier::unknown_order::CtOption;
 
-#[derive(Clone, ZeroizeOnDrop)]
+#[derive(Clone)]
 pub(crate) struct Private {
```

Comment:
Why were the `zeroize(skip)` deleted here?

Reply:
Same reason as above.